pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/WorkerServer.java

```diff
@@ -96,7 +136,6 @@ public void run() {
             log.error("ex: {}", ex, ex);
             final String message = getErrorMessage(server, this.workerConfig.getWorkerPort(), ex);
             log.error(message);
-            System.exit(1);
```

Comment:
@sijie integration test is failing because I have removed it in this PR. in order to test tls, we need to start worker-server and while stopping it in unit-test, it was throwing interruption exception on the thread which was causing system.exit in junit process. 
and it's also bad practice to do system.exit on exception. 
However, this change was causing integration test failure.
to fix it, we can either keep system.exit for now and disable tls-unit test or we have to fix integration test.
I wanted to refactor this method lil bit to fix it but we merge it before.

Reply:
@rdhabalia ah! sorry. I can revert it back ... does that work?

Reply:
that's fine. let me disable test for now.. will create a PR.

Reply:
created #2297 to fix the issue.